examples/maildev/index.js

```diff
@@ -12,11 +12,47 @@ module.exports = async function (fastify, options) {
   fastify.register(fastifyMail, { pov: { engine: { nunjucks } }, transporter })
 
   fastify.get('/sendmail', async (req, reply) => {
-    const recipients = ['test@example.com']
-    const templates = './templates'
-    const context = { name: 'Test Name', sender: 'sender@example.com' }
+    const { name } = req.query
+    const templatePath = './templates'
+    const context = { name: name || 'Test Name', sender: 'sender@example.com' }
```

Comment:
are you sitting on a commit here? i don't see any changes wrt to the resolved comments where the templatePath and context one liners are moved down under the message and into the opts const

mainly i was thinking that even though it's a fine example you've set up for the dynamic querystring passing that i'm not sure it's needed here to show off the template feature. so the simplicity for the user comes from simply having some static context passed into the templating system and we should be good (less docs/tests/supporting code we need to manage for our plugin)

Reply:
yes had missed the final git push. Friday-evening-itus :)